Meta/.shell_include.sh

```diff
@@ -20,9 +20,8 @@ die() {
 find_executable() {
   paths=("/usr/sbin" "/sbin")
 
-  if [ "$(uname -s)" = "Darwin" ]; then
-    paths+=("/usr/local/opt/e2fsprogs/bin" "/usr/local/opt/e2fsprogs/sbin")
-    paths+=("/opt/homebrew/opt/e2fsprogs/bin" "/opt/homebrew/opt/e2fsprogs/sbin")
+  if [ "$(uname -s)" = "Darwin" ] && [ -n "$HOMEBREW_PREFIX" ]; then
+    paths+=("$HOMEBREW_PREFIX/opt/e2fsprogs/bin" "$HOMEBREW_PREFIX/opt/e2fsprogs/sbin")
```

Comment:
```suggestion
  if [ "$(uname -s)" = "Darwin" ] && [ -n "${HOMEBREW_PREFIX}" ]; then
    paths+=("${HOMEBREW_PREFIX}/opt/e2fsprogs/bin" "${HOMEBREW_PREFIX}/opt/e2fsprogs/sbin")
```

If I remember my bash expansions correctly we could also compress this to the following:

```suggestion
  if [ "$(uname -s)" = "Darwin" ] && [ -n "${HOMEBREW_PREFIX}" ]; then
    paths+=("${HOMEBREW_PREFIX}/opt/e2fsprogs/{bin,sbin}")
```

In any case, can we be sure that `HOMEBREW_PREFIX` is always set?

Reply:
I'm also not sure if `paths+=("${HOMEBREW_PREFIX}/opt/e2fsprogs/{bin,sbin}")` will expand correctly, so I'll omit that for now.